src/server/common/model/answer/receiveMethod/receiveMethod.test.js

```diff
@@ -0,0 +1,94 @@
+import { ReceiveMethodAnswer } from './receiveMethod.js'
+
+const validReceiveMethodPayload = {
+  receiveMethod: 'email'
+}
+
+describe('#ReceiveMethod.validate', () => {
+  it('should return true for valid receiveMethod', () => {
+    const receiveMethod = new ReceiveMethodAnswer(validReceiveMethodPayload)
+    const { isValid } = receiveMethod.validate()
+
+    expect(isValid).toBe(true)
+  })
+
+  it('should return false for an empty input', () => {
+    const receiveMethod = new ReceiveMethodAnswer({
+      receiveMethod: 'invalid value'
+    })
+
+    const { isValid, errors } = receiveMethod.validate()
+
+    expect(isValid).toBe(false)
+    expect(errors.receiveMethod.text).toBe(
+      'Select how you would like this licence sent to you'
+    )
+  })
+
+  it('should return false for malformed input', () => {
```

Comment:
is that any different from the previous one with a string set to "invalid value"?